Assets/loadAudioSettings.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class loadAudioSettings : MonoBehaviour
```

Comment:
```suggestion
public class LoadAudioSettings : MonoBehaviour
```
Make sure to change filename too

Reply:
Will do. Should I move it to a specific script folder too?